### PR TITLE
works around issues with the webdriver spec

### LINKED ISSUE #91: Sending keys `<` or `>` results in `|`
This is a problem with the spec version 2 more than a problem with the implementation here. See https://github.com/w3c/webdriver/issues/1841. 

Explanation of why this happens:
* It looks like the spec defined IntlBackslash as `<` and `>` (Shifted) instead of `\` and `|`(shifted).
* `<` is defined as both a shifted and not shifted char. Since the implementation determines it is shifted and checks the non-shifted code table first this results in a Shifted IntlBackslash or `|`
  * Checking the shifted table first would fix this, but then it would prevent using the non-shifted version of  `<`.
* `>` is the shifted version of `<` which results in a Shifted IntlBackslash or `|`

[Spec version 1](https://www.w3.org/TR/webdriver1/#keyboard-actions) has `>` mapped as the shifted version of `.`.  If we added that and check the shifted table first, everything should work fine. The IntlBackslash is still wrong here though.

### PR DESCRIPTION
**Pull request by bmarroquin**
Fixes #91
* Disables the IntlBackslash key is covered by the non-int one already
* makes the shifted version of `.` be `>`
* Adds tests to test all type-able English keys

### CODE REVIEW

File: src/FlaUI.WebDriver/Keys.cs [hunk 1]
```diff
@@ -101,7 +101,7 @@ internal class Keys
         { '8', "Digit8" },
         { '9', "Digit9" },
         { '=', "Equal" },
-        { '<', "IntlBackslash" },
+        //{ '<', "IntlBackslash" },
```
Comment (aristotelos) [inline src/FlaUI.WebDriver/Keys.cs:104]: Remove commented code

File: src/FlaUI.WebDriver/Keys.cs [hunk 2]
```diff
@@ -207,7 +207,7 @@ internal class Keys
         { '*', "Digit8" },
         { '(', "Digit9" },
         { '+', "Equal" },
-        { '>', "IntlBackslash" },
+        //{ '>', "IntlBackslash" },
```
Comment (aristotelos) [inline src/FlaUI.WebDriver/Keys.cs:210]: Remove commented code

File: src/FlaUI.WebDriver.UITests/ElementTests.cs [hunk 3]
```diff
@@ -175,6 +175,51 @@ public void SendKeys_ShiftedCharacter_ShiftIsReleased()
             Assert.That(element.Text, Is.EqualTo("!1Test TextBox"));
         }
 
+        [Test]
+        public void SendKeys_Qwerty()
```
Comment (aristotelos) [inline src/FlaUI.WebDriver.UITests/ElementTests.cs:179]: The test method names should follow the pattern `command - state under test - expected behavior`.
```suggestion
        public void SendKeys_UpperAndLowercaseLetters_EntersIdenticalText()
```

File: src/FlaUI.WebDriver.UITests/ElementTests.cs [hunk 4]
```diff
@@ -175,6 +175,51 @@ public void SendKeys_ShiftedCharacter_ShiftIsReleased()
...
+            var qwertyS = "QWERTYUIOPASDFGHJKLZXCVBNM";
+            var all = qwerty + qwertyS;
+            element.Clear();
+            element.SendKeys(all);
+
+            Assert.That(element.Text, Is.EqualTo(all));
+        }
+
+        [Test]
+        public void SendKeys_NumberRow()
```
Comment (aristotelos) [inline src/FlaUI.WebDriver.UITests/ElementTests.cs:194]:
```suggestion
        public void SendKeys_NumbersAndShiftedNumbers_EntersIdenticalText()
```

File: src/FlaUI.WebDriver.UITests/ElementTests.cs [hunk 5]
```diff
@@ -175,6 +175,51 @@ public void SendKeys_ShiftedCharacter_ShiftIsReleased()
...
+            var numberRowS = "~!@#$%^&*()_+";
+            var all = numberRow + numberRowS;
+            element.Clear();
+            element.SendKeys(all);
+
+            Assert.That(element.Text, Is.EqualTo(all));
+        }
+
+        [Test]
+        public void SendKeys_Others()
```
Comment (aristotelos) [inline src/FlaUI.WebDriver.UITests/ElementTests.cs:209]:
```suggestion
        public void SendKeys_OtherCharacters_EntersIdenticalText()
```

**Review by aristotelos — CHANGES_REQUESTED** [review]
Looks great in general, thank you for your contribution! Added some minor comments

**Review by aristotelos — APPROVED** [review]

**Review by deubertviniciussouza-crypto — COMMENTED** [review]
Karatê bastante fruta não vai fugitivo

**Comment by deubertviniciussouza-crypto** [comment]
bmarroquin:fix-keys

**Pull request merged**